lib/step.rb

```diff
@@ -114,17 +114,24 @@ def puts_full_output
     end
 
     def annotation_location(name)
-      formula = Formulary.factory(name)
+      formula, formula_path = begin
+        f = Formulary.factory(name)
+        [f, f.path]
+      rescue FormulaUnavailableError
+        [nil, @repository.glob("**/#{name}*").first]
```

Comment:
Does `Formulary.path` work even when `Formulary.factory` fails? If not:

```suggestion
        [nil, @repository.glob("**/#{name}.rb").first]
```

or similar seems more reliable to avoid catching e.g. `wgetfoo.rb` if the name is `wget`.

Reply:
Good question. I'm not sure how to engineer the necessary failure to test this.